# singlecell_workshop
Single-cell and Spatial data analyses

# Topics
## Single-cell transcriptomics:
QC<br>
Batch-effect correction<br>
Clustering<br>
Automated annotation<br>
Compositional changes<br>
Downstream analyes<br>

## Spatial transcriptomics:
Data loading, processing and analysis:<br>
[Spatial transcriptomics (10X Genomics)](https://github.com/hayatlab/singlecell_workshop/blob/main/03_Visium_Heart_Workshop_GCB23.ipynb "Part_03")<br>

## Single-cell Spatial transcriptomics:
Introduction to high-resolution spatial techniques such as Xenium, and MERFISH:<br>
[Single-cell spatial transcriptomics (MERFISH)](https://github.com/hayatlab/singlecell_workshop/blob/main/04_Vizgen_Liver_Workshop_GCB23.ipynb "Part_04")<br>
[Single-cell spatial transcriptomics (Xenium)](https://github.com/hayatlab/singlecell_workshop/blob/main/04_Xenium_Kidney_Workshop_GCB23.ipynb "Part_04")<br>


# Dataset
We will use our published multiomics single-cell dataset (Kuppe et al., Nature 2022).<br>
The dataset is provided here: https://zenodo.org/record/6578047/files/snRNA-seq-submission.h5ad?download=1

A sub-sampled dataset is provided here: https://drive.google.com/drive/folders/1y_-TyG4O-9ii2pk8Nw2ySwQm1eNQlg3S?usp=drive_link

# Organizers
Tore Bleckwehl, PhD, Post-doc, UniKlinik RWTH, Aachen<br>
Sikander Hayat, PhD, Group leader in Translational Data Science, UniKlinik RWTH, Aachen
